docs/admin-guide/admin-configuration.md

```diff
@@ -80,10 +82,14 @@ The following features are configured under the `extensions` attribute.
 -   [Search](#search_config)
 -   [Scrub](#scrub_config)
 -   [Lint](#lint_config)
+-   [TrustImage](#trust_config)
+-   [UI](#ui_config)
   
 An extension feature is usually enabled by the presence of the feature’s attribute under `extensions`. An extension feature can then be disabled by either omitting the feature attribute or by including an `enable` attribute with a value of `false`.
 
-> :pencil2: An additional extension feature, [User Preferences](#userprefs_config), is enabled when the Search feature is enabled, and is not configured under the `extensions` section.
+> :pencil2: Two API-only extensions, [User Preferences](#userprefs_config) and [Mgmt](#mgmt_config), are automatically enabled when the `Search` and `UI` extensions are enabled. These API-only extensions are not configured under the `extensions` section of the configuration file.
```

Comment:
Since the mgmt API would also be used for ZLI to get the server version, we will probably change the mgmt API to be enabled only based on the search being enabled.
This is not yet merged, the PR is work in pogress.

@rchincha ^^

Reply:
Also thinking the same for userprefs. Since the UI can be run remotely as a separate service, why should we enable the embedded UI in order for the remote UI to work. This use case escaped me when I coded the conditions.

Reply:
UI running as a separate service is only for dev environments
"version" could simply be a field in our Config object

Reply:
What about the clustering case? Is every zot supposed to have a separate UI?

Reply:
> UI running as a separate service is only for dev environments "version" could simply be a field in our Config object

The Config object which would be available in an API used by CLI but which is only available if UI is enabled, that is an issue.

Reply:
Extensions section of Admin Config Guide now says:

Mgmt is enabled when the `Search` extension is enabled.
User Preferences is enabled when both the `Search` and `UI` extensions are enabled.